src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { classNames } from "../../utils";
+import SliderHandle from "./SliderHandle.jsx";
+import useSlider from "./hooks";
+import styles from "./styles.css";
+
+const Slider = ({
+  double,
+  disabled,
+  axis,
+  min,
+  max,
+  step,
+  handles,
+  ...props
+}) => {
+  const {
+    container,
+    handleTrackMouseDown,
+    valueStyle,
+    handleHooks
+  } = useSlider(double, disabled, axis, min, max, step, handles);
+
+  return (
+    <div
+      {...props}
+      ref={container}
+      className={classNames(
+        styles[axis],
+        styles.track,
+        disabled && styles.disabled
+      )}
+      onTouchStart={handleTrackMouseDown}
+      onMouseDown={handleTrackMouseDown}>
+      {double && (
+        <SliderHandle
+          ref={handleHooks[1].handle}
```

Comment:
as you access `handleHooks[1]` & `handleHooks[0]` please consider defining a const for each one of them maybe with a bit more meaningful name